### PR TITLE
chore(linter): fix non-deterministic snapshots for convert tslint to …

### PR DESCRIPTION
**Pull request by FrozenPandaz**
…eslint

## Current Behavior

The `convert-tslint-to-eslint` rules started failing because something changed with how `tslint-to-eslint-config` converts the `no-console` rule.

## Expected Behavior

The conversion process isn't deterministic across different node versions so we'll just check that it at least includes `console.log`

## Related Issue(s)

Fixes #

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -146,7 +146,7 @@ jobs:
       NX_E2E_CI_CACHE_KEY: e2e-circleci-<< parameters.os >>
       SELECTED_PM: << parameters.pm >>
       NX_E2E_RUN_CYPRESS: 'false'
-      NX_VERBOSE_LOGGING: 'true'
```
Comment (meeroslav) [inline .circleci/config.yml:149]: This should be done in a separate PR I believe, best to just rollback @vsavkin's commit

**Review by AgentEnder — APPROVED** [review]

**Review by meeroslav — APPROVED** [review]
